ubuntu/debian/control

```diff
@@ -22,8 +22,10 @@ Pre-Depends: ${misc:Pre-Depends}
 XB-Ruby-Versions: ${ruby:Versions}
 Depends: ruby | ruby-interpreter,
          ${misc:Depends}
-Breaks: ignition-tools
-Replaces: ignition-tools
+Breaks: ignition-tools,
```

Comment:
Ideally we'd remove the breakage of `ignition-tools`, that was done just so we could iterate faster while both were still installing `ign`.

My initial inclination would be to:

1. User tries to install `gz-tools2`
2. If `ignition-tools` is installed, don't install the `ign` symlink to `gz`
3. If `ignition-tools` isn't installed, install the symlink - but then if the user tries to install `ignition-tools` afterwards, it would be nice to remove that symlink and let the user know

Reply:
Hmmm, I think it'll need a companion set of PRs for `ignition-tools` and the corresponding release repo

There are some small complications:
- The flow for deb install is:
  - CMake install() -> file gets put into the CMake install space -> deb packaging takes the install space file and packages it into the debian file
- As I understand it, debian package management control gives us the tools to specify that it's ok to overwrite files, but **not** to fail gracefully if files aren't installed on the CMake side.

I think there's a way to achieve what we want, but it'll be a bit janky.

- User tries to install gz-tools2
- If ignition-tools is installed, don't install the ign symlink to gz
  - If we don't install() the symlink, we can't put the install directive into the debian control file.
  - So most likely what I will do, is try to **copy paste the `ignition-tools` `ign` executable** to the ignition-tools2 CMake install space and package that instead of creating the symlink if `ignition-tools_FOUND`
- If ignition-tools isn't installed, install the symlink
  - This is fine
- but then if the user tries to install ignition-tools afterwards, it would be nice to remove that symlink and let the user know
  - This will be achieved by installing the symlink in the ignition-tools2 install space, and overwriting the pre-existing one as specified in the debian control file

This will need:
- Breaks and Replaces directives on `control` to allow overwriting files for:
  - on `ignition-tools`: `ignition-tools2`
  - on `ignition-tools2`: `ignition-tools`

Reply:
Thanks for documenting all this, let's go with the current implementation for now and revisit later.